README.md

```diff
@@ -107,6 +107,19 @@ API](https://developers.google.com/google-ads/api/docs/start).
     the examples require parameters. You can see what are required by running
     code examples with `--help` as a command-line parameter.
 
+## Running in a Docker container
```

Comment:
Can you move this to be just before "Logging"?

Reply:
Done.